src/Modules/Data/Commands.Data/help/Get-PowerBIDataset.md

```diff
@@ -127,7 +128,7 @@ Accept wildcard characters: False
 ```
 
 ### -Scope
-Indicates scope of the call. Individual returns only datasets assigned to the caller; Organization returns all datasets within a tenant (must be an administrator to initiate). Individual is the default.
+Indicates scope of the call. Individual returns datasets from "My Workspace" unless -Workspace or -WorkspaceId is specified which shows datasets under the workspace assigned to the caller; Organization returns all datasets within a tenant (must be an administrator to initiate). Individual is the default.
```

Comment:
This is confusingly worded. Is this true to what you had in mind?
Indicates scope of the call. Individual returns datasets from "My Workspace" by default. Using the -Workspace or -WorkspaceId parameter, you can show datasets from any workspace assigned to the caller.

Reply:
I'm going to change it to this:
Indicates scope of the call. Individual returns datasets from "My Workspace" by default. With -Workspace or -WorkspaceId, datasets under the workspace assigned to the caller are returned; Organization returns all datasets within a tenant (must be an administrator to initiate). Individual is the default.